### ISSUE TITLE
Wavelength dependant loss in fibers

### ISSUE THREAD

**Issue by Fredrik-Smartoptics** [Telecominfraproject/oopt-gnpy#318]
In all fibertypes there is a wavelength dependant loss (WDL). In the C-band higher frequencies are attenuated more than lower frequencies. This WDL amounts to about 0,8 dB/100 km difference over the entire C-band, hence I would not consider it negligable especially considering simulations with many spans. Fiber WDL is very easily verified by just measuring loss through a fiber at a low frequency (like 191.30 THz) and one at a high frequency (like 196.40 THz). This difference in loss has an impact on OSNR and also non-linear effects, since some channels are attenuated quicker than others.

Looking through GNPy I cannot see that this WDL is taken into account. The loss seem to be uniform for all channels. I believe the results of GNPy can be further improved if the fiber WDL is taken into account.

**Comment by jktjkt** [comment]
Thanks for reporting this, Fredrik. We took a short break this week after the TIP Summit, so I have not got a chance to talk to our optical scientists whether the GN model accounts for WDL or not. I agree with you that this does not appear to be taken into account based on the current version of the code.

**Comment by jktjkt** [comment]
It turns out that the WDL is being taken into account when using the Raman-aware propagation model (i.e., the `--sim-params` option). The price for this feature is quite a noticeable slowdown due to the increased complexity of the calculations.

We discussed whether it makes sense to take into account the WDL on its own without considering SRS and the energy transfer from higher-frequency λ to the longer wavelengths. I'm afraid that the answer is unclear at this point. So, we *could* add WDL-awareness into the non-Raman propagation model, but we don't know whether doing that would improve actual real-world accuracy. I think we will have to revisit this next week -- please stay tuned. If you happen to have a reference to a published paper (or anything, really) which compares contributions of SRS and WDL under varying conditions, I'm sure that our colleagues from Polito would love to hear about that.

Cc: @aleFerrari

**Comment by jktjkt** [comment]
A small update -- we've confirmed that we're going to look into WDL even without SRS being active. Thanks for suggesting this once again.

**Comment by caffery-chen** [comment]
Dear jktjkt, 

not sure if the WDL here you mentioned is the concept as same as the linear regime WDL mentioned in the paper "Experimental Investigation of Stimulated Raman Scattering Limitation on WDM Transmission Over Various Types of Fiber Infrastructures" by Sebastien Bigo.

It turns out that the tilt needs to consider both linear tilt and also the SRS tilt altogether.

Please let me know your thougts.

Thanks.

**Comment by jktjkt** [comment]
@caffery-chen , I cannot answer this one, but hopefully @AndreaDAmico or someone else from the POLITO team can.

**Comment by AndreaDAmico** [comment]
@caffery-chen , after a quick look at the mentioned paper, my understanding is that the WDL concepts are the same. The peculiarity of the linear regime (low power) is that there is not any significant SRS effect, as the latter is proportional to the power. 
In the current version, both the SRS and the WDL are properly considered only in the `RamanFiber`. 
As for the SRS, also the WDL is addressed within the standard `Fiber` in my recent work. 
In general, the WDL profile is considered flat if a specific profile is not provided. Up to now we have not found a better default choice as the WDL profile can largely vary depending on the specific fiber under consideration.

**Comment by caffery-chen** [comment]
Dear @AndreaDAmico, 

Is this WDL in linear regime due to the wavelength dependent fiber attenuation within C Band?
If so, I was surprised that this dependency is often ignored because it is proportional to transmission length and will ramp up significantly.

Thanks.

**Comment by Fredrik-Smartoptics** [comment]
@caffery-chen @jktjkt @AndreaDAmico 

I agree with Andrea. Fiber WDL is often overlooked (=assumed 0) and is one of the most limiting factors in long haul transmission with ILAs (in line amplifiers). Of course if WSS are used like in ROADM sites the compensation can be done perfectly. But using just EDFA amplifier tilt to compensate is far from perfect. The amplifiers tilt will have flaws that are not random, specifically channels around 194 THz often suffers from low gain in comparison to the others. With many ILA spans the penalty for these channels gets pronounced.
My experience is that WDL of fibers is very predictable and does not vary much between suppliers or even fibertypes. I cannot say that I have done a scientific analysis but all the fibers I measured in my career have had tilt of about 0,008 dB/km over the C-band. Also during fault analysis in customers network over many spans the measured OSNR profile over channels can be matched to a very high degree by using the fixed fiber WDL in the calculation. Using 0 as fiber WDL gives very different results that does not match. I admit I have not measured or analyzed many non-G652 fibers. When speaking to component experts what I have been told is that the reason for the fiber tilt lies in the properties of Silica. That they say is the reason why it is similar in all fibers.

**Comment by Fredrik-Smartoptics** [comment]
I also want to add that as discussed above there are two components of this. The linear WDL attenuation and the consequence of different attenuation between channels for non-linear effects. Maybe GNPy can consider only the linear WDL and direct OSNR effect as a start? To simplify the calculations.

**Comment by AndreaDAmico** [comment]
@caffery-chen,@Fredrik-Smartoptics  

> Is this WDL in linear regime due to the wavelength dependent fiber attenuation within C Band?
> If so, I was surprised that this dependency is often ignored because it is proportional to transmission length and will ramp up significantly.

Even if GNPy provide the evaluation of the GSNR for each channel in a WDM comb (C-Band with the possibility to extend it to other bands), the simulation engine was focused on the central channel, which is usually the worst case scenario, for planning purposes. In this perspective, if the user provides the WDL value evaluated at the central frequency, GNPy provides a very accurate estimation of the power evolution of the central channel, as the SRS effect over the central channel is negligible. 
Of course the same cannot be said for side channels, but form the central channel point of view, this would affect only the nonlinear interference (NLI) noise. This inaccuracy is not significant in the final evaluation of the GSNR, as the impairment due to the NLI is not the main contribution to the total signal degradation, being half of the ASE noise in optimal condition.

> I also want to add that as discussed above there are two components of this. The linear WDL attenuation and the consequence of different attenuation between channels for non-linear effects. Maybe GNPy can consider only the linear WDL and direct OSNR effect as a start? To simplify the calculations.

Recently, we are integrating more accurate implementations in order to obtain a more precise evaluation for all the channel within the WDM comb without a significant increasing of the computational time. These implementation include the SRS and the WDL.
 
> I agree with caffery-chen. Fiber WDL is often overlooked (=assumed 0) and is one of the most limiting factors in long haul transmission with ILAs (in line amplifiers). Of course if WSS are used like in ROADM sites the compensation can be done perfectly. But using just EDFA amplifier tilt to compensate is far from perfect. The amplifiers tilt will have flaws that are not random, specifically channels around 194 THz often suffers from low gain in comparison to the others. With many ILA spans the penalty for these channels gets pronounced.
> My experience is that WDL of fibers is very predictable and does not vary much between suppliers or even fiber types. I cannot say that I have done a scientific analysis but all the fibers I measured in my career have had tilt of about 0,008 dB/km over the C-band. Also during fault analysis in customers network over many spans the measured OSNR profile over channels can be matched to a very high degree by using the fixed fiber WDL in the calculation. Using 0 as fiber WDL gives very different results that does not match. I admit I have not measured or analyzed many non-G652 fibers. When speaking to component experts what I have been told is that the reason for the fiber tilt lies in the properties of Silica. That they say is the reason why it is similar in all fibers.

In  _A. D’Amico et al., "Enhancing Lightpath QoT Computation With Machine Learning in Partially Disaggregated Optical Networks," in IEEE Open Journal of the Communications Society, vol. 2, pp. 564-574, 2021, doi: 10.1109/OJCOMS.2021.3066913_, we show some examples of measured WDL for different fibers. In this case, significant variations of the WDL are present. 
In _Walker,  S.  “Rapid  modeling  and  estimation  of  total spectral loss in optical fibers” JLT 4.8 (1986): 1125-1131_, the author extensively describes this fiber characteristic.

**Comment by Fredrik-Smartoptics** [comment]
@AndreaDAmico, thanks I will check out the publication.

**Comment by caffery-chen** [comment]
@AndreaDAmico @Fredrik-Smartoptics 
Thank you for the discussion, the experience you shared is very valuable. 
I will compare the modeling and real measurements in the near future and come back to this thread.

**Comment by Fredrik-Smartoptics** [comment]
Dear @AndreaDAmico 

I find your article very interesting and well written. Regarding fiber WDL I cannot seem to find measurement results although I might very well have missed it. The level of detail is high which is a good thing of course. I do find average alfa in table 1. Could you help me out?

**Comment by AndreaDAmico** [comment]
@Fredrik-Smartoptics 
You are right, there is only the average alpha reported in Table 1, not the measured WDL profile. In any case, in Chapter V, there is the description of the model that can be used to characterise the WDL for the specific fiber. What we obtain is a varying tilt, depending on the considered fiber.  We published the measured WDL in an accepted OFC paper but I think this is not accessible yet, and I am not sure how to share them.

**Comment by Fredrik-Smartoptics** [comment]
@AndreaDAmico 
Thanks for the update. Do you know when it will be accessible/possible to share. If not too long I can wait for that.

**Comment by EstherLerouzic** [comment]
solved with v2.8
